resources/views/glossary.blade.php

```diff
@@ -24,6 +24,26 @@
                             @endif
                         </div>
                         <p class="mt-2">{{ $term->getTranslation('description', locale()) }}</p>
+                        @if (count($term->resourcesForUser())>0)
```

Comment:
Spaces around `>` please

```suggestion
                        @if (count($term->resourcesForUser()) > 0)
```